Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1162,6 +1164,7 @@ def lhegensim(fragment,howMuch):
 steps['RECODR2_25nsreHLT']=merge([{'--hltProcess':'reHLT'},steps['RECODR2_25ns']])
 steps['RECODR2_50nsreHLT']=merge([{'--hltProcess':'reHLT'},steps['RECODR2_50ns']])
 steps['RECODR2_2016reHLT']=merge([{'--hltProcess':'reHLT','--conditions':'auto:run2_data_relval'},steps['RECODR2_2016']])
+steps['RECODR2newL1repack_2016reHLT']=merge([{'-s':'L1REPACK:FullSimTP,RAW2DIGI,L1Reco,RECO,EI,PAT,ALCA:SiStripCalZeroBias+SiStripCalMinBias+TkAlMinBias+EcalESAlign,DQM:@standardDQM+@miniAODDQM','--hltProcess':'reHLT'},steps['RECODR2_2016']])
```

Comment:
@mmusich Not exactly.  My worry was  that the step3 defined for a workflow ```'RECODR2_2016reHLT'``` defined above as:
<pre>
 steps['RECODR2_2016reHLT']=merge([{'--hltProcess':'reHLT','--conditions':'auto:run2_data_relval'},steps['RECODR2_2016']])
</pre>
had explicitly used ```'--conditions':'auto:run2_data_relval'``` in the definition, while that was not the case for the workflow ```'RECODR2newL1repack_2016reHLT'```.

Judging by the output of your command ```runTheMatrix.py -nel 136.7321``` the same flag  ```'--conditions':'auto:run2_data_relval'``` is picked up in step3 for the later workflow, so I guess we are fine.